packages/sst/src/stacks/deploy.ts

```
@@ -248,12 +250,16 @@ async function addInUseExports(
-    path.resolve(cdkToolkitPath, "..", "api", "deployments", "deployments.js")
+    pathToFileURL(
+      path.resolve(cdkToolkitPath, "..", "api", "deployments", "deployments.js")
+    ).href
   );
   const { IoHelper } = await import(
-    path.resolve(cdkToolkitPath, "..", "api", "io", "private", "io-helper.js")
+    pathToFileURL(
+      path.resolve(cdkToolkitPath, "..", "api", "io", "private", "io-helper.js")
+    ).href
   );
   const provider = await useAWSProvider();
```

Biome (2.1.2)

[error] This hook is being called indirectly and conditionally, but all hooks must be called in the exact same order in every component render.

This is the call path until the hook.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)